server/datastore/mysql/hosts_test.go

```diff
@@ -8689,8 +8689,11 @@ func testHostHealth(t *testing.T, ds *Datastore) {
 	require.NoError(t, ds.LoadHostSoftware(context.Background(), h, false))
 
 	soft1 := h.Software[0]
-	if soft1.Name != "bar" {
-		soft1 = h.Software[1]
+	for _, item := range h.Software {
```

Comment:
doesn't seem like this host object will contain the updated software.  do we need to call ds.Host again before this loop?

Reply:
`LoadHostSoftware` fetches the software and adds it to the host object.